[2024-03-10T23:21:42.438+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=train_penguin_sex_model/run_id=manual__2024-03-10T23:21:39.451814+00:00/task_id=train_model permission to 509
[2024-03-10T23:21:42.439+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=train_penguin_sex_model/run_id=manual__2024-03-10T23:21:39.451814+00:00/task_id=train_model permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=train_penguin_sex_model/run_id=manual__2024-03-10T23:21:39.451814+00:00/task_id=train_model'
[2024-03-10T23:21:42.480+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=train_penguin_sex_model/run_id=manual__2024-03-10T23:21:39.451814+00:00/task_id=train_model permission to 509
[2024-03-10T23:21:42.482+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=train_penguin_sex_model/run_id=manual__2024-03-10T23:21:39.451814+00:00/task_id=train_model permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=train_penguin_sex_model/run_id=manual__2024-03-10T23:21:39.451814+00:00/task_id=train_model'
[2024-03-10T23:21:42.531+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_penguin_sex_model.train_model manual__2024-03-10T23:21:39.451814+00:00 [queued]>
[2024-03-10T23:21:42.553+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_penguin_sex_model.train_model manual__2024-03-10T23:21:39.451814+00:00 [queued]>
[2024-03-10T23:21:42.554+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2024-03-10T23:21:42.580+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): train_model> on 2024-03-10 23:21:39.451814+00:00
[2024-03-10T23:21:42.593+0000] {standard_task_runner.py:57} INFO - Started process 120 to run task
[2024-03-10T23:21:42.597+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'train_penguin_sex_model', 'train_model', 'manual__2024-03-10T23:21:39.451814+00:00', '--job-id', '149', '--raw', '--subdir', 'DAGS_FOLDER/train_model.py', '--cfg-path', '/tmp/tmpvmdsc96n']
[2024-03-10T23:21:42.600+0000] {standard_task_runner.py:85} INFO - Job 149: Subtask train_model
[2024-03-10T23:21:42.757+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=train_penguin_sex_model/run_id=manual__2024-03-10T23:21:39.451814+00:00/task_id=train_model permission to 509
[2024-03-10T23:21:42.760+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=train_penguin_sex_model/run_id=manual__2024-03-10T23:21:39.451814+00:00/task_id=train_model permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=train_penguin_sex_model/run_id=manual__2024-03-10T23:21:39.451814+00:00/task_id=train_model'
[2024-03-10T23:21:42.765+0000] {task_command.py:410} INFO - Running <TaskInstance: train_penguin_sex_model.train_model manual__2024-03-10T23:21:39.451814+00:00 [running]> on host f95f5ac280e4
[2024-03-10T23:21:42.960+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='train_penguin_sex_model' AIRFLOW_CTX_TASK_ID='train_model' AIRFLOW_CTX_EXECUTION_DATE='2024-03-10T23:21:39.451814+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-10T23:21:39.451814+00:00'
[2024-03-10T23:21:43.238+0000] {taskinstance.py:1847} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/train_model.py", line 38, in train_penguin_sex_model
    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)
  File "/home/airflow/.local/lib/python3.7/site-packages/sklearn/model_selection/_split.py", line 2421, in train_test_split
    n_samples, test_size, train_size, default_test_size=0.25
  File "/home/airflow/.local/lib/python3.7/site-packages/sklearn/model_selection/_split.py", line 2101, in _validate_shuffle_split
    "aforementioned parameters.".format(n_samples, test_size, train_size)
ValueError: With n_samples=0, test_size=0.2 and train_size=None, the resulting train set will be empty. Adjust any of the aforementioned parameters.
[2024-03-10T23:21:43.267+0000] {taskinstance.py:1373} INFO - Marking task as UP_FOR_RETRY. dag_id=train_penguin_sex_model, task_id=train_model, execution_date=20240310T232139, start_date=20240310T232142, end_date=20240310T232143
[2024-03-10T23:21:43.300+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 149 for task train_model (With n_samples=0, test_size=0.2 and train_size=None, the resulting train set will be empty. Adjust any of the aforementioned parameters.; 120)
[2024-03-10T23:21:43.346+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 1
[2024-03-10T23:21:43.392+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
